pkg/kubelet/metrics/collectors/resource_metrics.go

```diff
@@ -54,7 +54,21 @@ var (
 		metrics.ALPHA,
 		"")
 
-	resouceScrapeResultDesc = metrics.NewDesc("scrape_error",
+	podCPUUsageDesc = metrics.NewDesc("pod_cpu_usage_seconds_total",
```

Comment:
How is this different from container_cpu_usage_seconds_total{pod="X",namespace="Y",container=""} (which should be total CPU seconds of the pod cgroup)?

Reply:
Not all processes associated with running a pod are within the container cgroups themselves, but instead within the pod cgroup created by kubelet. For example, conmon in the standard/crio case.

When looking at sandboxed runtimes, this is even more significant, as the pod cgroup will include processes like the VMM, vCPU threads, IO threads, etc.